Formula/rio.rb

```diff
@@ -0,0 +1,24 @@
+class Rio < Formula
+  desc "Kubernetes based MicroPaaS"
+  homepage "https://rio.io"
+  url "https://github.com/rancher/rio.git",
+    :using    => :git,
+    :tag      => "v0.1.1-rc3",
+    :revision => "2771703b180bfe316567d6bc7242cf5490c8358f"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/rancher/rio"
+    dir.install buildpath.children
+    cd dir
+
+    system "scripts/build"
+    bin.install "bin/rio"
+  end
+
+  test do
+    system "#{bin}/rio", "--version"
```

Comment:
This test is insufficient. Please test actual functionality. You can parse logs & stdout/stderr for suitably generic text, check the return status of actual commands and check for the creation of files.

Reply:
Copied the test from `kubernetes-cli` there isn't much more to test that doesn't require talking to a Kubernetes cluster.

Reply:
@rickard-von-essen:

Hmmm. No local logging when you try to talk to one that's not there?

Catching an error that confirms it's trying to do what you asked it to is better than just parsing boilerplate strings from the output.

> Copied the test from `kubernetes-cli`

Just because am old formula has a bad test, doesn't mean we want to add new ones with bad tests 😉